test/e2e/framework/pv_util.go

```diff
@@ -857,6 +857,47 @@ func MakePod(ns string, nodeSelector map[string]string, pvclaims []*v1.Persisten
 	return podSpec
 }
 
+// Returns a pod definition based on the namespace using nginx image
+func MakeNginxPod(ns string, nodeSelector map[string]string, pvclaims []*v1.PersistentVolumeClaim) *v1.Pod {
+	podSpec := &v1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Pod",
+			APIVersion: testapi.Groups[v1.GroupName].GroupVersion().String(),
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "pvc-tester-",
+			Namespace:    ns,
+		},
+		Spec: v1.PodSpec{
+			Containers: []v1.Container{
+				{
+					Name:  "write-pod",
+					Image: "nginx",
```

Comment:
Other e2e code is using `imageutils.GetE2EImage(imageutils.NginxSlim)` to define the nginx image

Reply:
please note - changing the image name here is non-trivial amount of work. I will have to go back to a kube version where these tests are Red and then make the image change, deploy a cluster and test it.

And then - I will have to come back to kube version where these tests are green. Deploy a new cluster and test it again.  

I am happy to change this if you insist, but it might be awhile before I get to it.

Reply:
Also - ideally each test has to run for couple of hours at least to make sure, there are no flakes.